src/components/app.jsx

```diff
@@ -1,19 +1,73 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import LocationDetails from './location-details';
+import ForecastSummaries from './forecast-summaries';
+import ForecastDetails from './forecast-details';
+import '../styles/app.scss';
+import SearchForm from './search-form';
 
-const App = props => (
-  <LocationDetails
-    city={props.location.city}
-    country={props.location.country}
-  />
-);
-
-App.propTypes = {
-  location: PropTypes.shape({
-    city: PropTypes.string,
-    country: PropTypes.string,
-  }).isRequired,
-};
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedDate: 0,
+      forecasts: [],
+      location: {
+        city: '',
+        country: '',
+      },
+    };
+    this.handleForecastSelect = this.handleForecastSelect.bind(this);
+    this.handleCitySearch = this.handleCitySearch.bind(this);
+  }
+
+  handleForecastSelect(date) {
+    this.setState({ selectedDate: date });
+  }
+
+  getData(url) {
+    fetch(url)
+      .then(response => response.json())
+      .then(data => this.setState({
+        location: {
+          city: data.location.city,
+          country: data.location.country,
+        },
+        forecasts: data.forecasts,
+      }));
+  }
+
+  componentDidMount() {
+    const url = 'https://mcr-codes-weather.herokuapp.com/forecast?city=London';
+    this.getData(url);
+  }
+
+  handleCitySearch(city) {
+    const newCity = city;
+    const url = `https://mcr-codes-weather.herokuapp.com/forecast?city=${newCity}`;
+    this.getData(url);
```

Comment:
Another reason to store you api url in a config file. You could reference the same variable here and avoid writing the same line of code again.